### PR TITLE
PvP Stats for Legends 6+ I have :)

### PR DESCRIPTION
**Pull request by gabrielglbh**

### CODE REVIEW

File: common/data/details.js [hunk 1]
```diff
@@ -22152,6 +22152,33 @@ window.details = {
             },
         ],
         potentialNotes: "#{enrage}",
+        festAbility: [
```
Comment (ThePieBandit) [inline common/data/details.js:22155]: This file isn't used anymore for PvP data. It will be removed soon. Instead, enter PVP data in rumble.json

Comment (gabrielglbh) [reply common/data/details.js:22155]: Got it. Should have noticed. I will double check with the units that have already the data on the JSON file and add the ones that dont have it in the coming days.

**Comment by Solaris333** [comment]
I didn't catch this until much later on, so some of the things in the rumble json has changed, so the PR is conflicting. I can manually add in the changes if you let me know which unit IDs you added or if you would like to resolve the conflicts, that works too.

**Comment by gabrielglbh** [comment]
I have tried several to insert them myself but I cannot get it right, the diff is just a mess. The units I added are 1413 (STR Whitebeard 6+), 1543 (DEX Ussop 6+), 1593 (PSY Gear 4 Luffy 6+), 1619 (INT Rayleigh 6+), 1764 (QCK Aokiji 6+), 1816 (DEX Ace 6+), 1921 (DEX Zoro 6+), 1922 (QCK Law 6+), 1928 (QCK Barto 6+), 2245 (INT Sanji 6+), 2357 (QCK Cavendish 6+), 2363 (STR Snakeman 6+), 2372 (STR Inu 6+), 2373 (PSY Neko 6+), 2444 (INT Doffy 6+), 2505 (INT Fujitora 6+), 2681 (STR Bullet 6+), 2588 (DEX Nami 6+), 2631 (PSY Shirahoshi 6+), 2739 (INT Katakuri 6+), 2830 (INT Robin 6+) and 2909 (QCK Whitebeard 6+).

Find attached the units in updated json format for you to just paste them in: [6-plus.txt](https://github.com/optc-db/optc-db.github.io/files/6064493/6-plus.txt)

**Pull request closed without merging**